components/application-configuration/ui/src/main/resources/PhenoTips/MeasurementsConfigurationEditor.xml

```diff
@@ -0,0 +1,386 @@
+      </cache>
+      <code>
+        <disabled>0</disabled>
+        <name>code</name>
+        <number>2</number>
+        <prettyName>Code</prettyName>
+        <rows>20</rows>
+        <size>50</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.TextAreaClass</classType>
+      </code>
+      <name>
+        <disabled>0</disabled>
+        <name>name</name>
+        <number>1</number>
+        <prettyName>Name</prettyName>
+        <size>30</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
+      </name>
+      <parse>
+        <disabled>0</disabled>
+        <displayFormType>select</displayFormType>
+        <displayType>yesno</displayType>
+        <name>parse</name>
+        <number>4</number>
+        <prettyName>Parse content</prettyName>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.BooleanClass</classType>
+      </parse>
+      <use>
+        <cache>0</cache>
+        <disabled>0</disabled>
+        <displayType>select</displayType>
+        <multiSelect>0</multiSelect>
+        <name>use</name>
+        <number>3</number>
+        <prettyName>Use this extension</prettyName>
+        <relationalStorage>0</relationalStorage>
+        <separator> </separator>
+        <separators>|, </separators>
+        <size>1</size>
+        <unmodifiable>0</unmodifiable>
+        <values>currentPage|onDemand|always</values>
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+      </use>
+    </class>
+    <property>
+      <cache>long</cache>
+    </property>
+    <property>
+      <code>var PhenoTips = (function (PhenoTips) {
+  var widgets = PhenoTips.widgets = PhenoTips.widgets || {};
+  widgets.MeasurementsConfigurationEditor = Class.create({
+
+    _ENHANCED_TOOL_CSS_CLASS : 'initialized',
+    _NEW_TOOL_SELECTOR: ':not(.initialized)',
+    
+    initialize : function (element) {
+      if (!element) return;
+      this._element = element;
+      this._form = element.up('form');
+      this._enhanceExtensionTools();
+      this._enableFieldDragDrop = this._enableFieldDragDrop.bind(this);
+      this._syncDefaultValues = this._syncDefaultValues.bind(this);
+      this._syncDefaultValues();
+    },
+
+    _enhanceExtensionTools : function () {
+      this._silenceMoveButtons();
+      this._enableFieldDragDrop();
+    },
+
+    _silenceMoveButtons : function () {
+      this._element.select('a.move' + this._NEW_TOOL_SELECTOR).invoke('observe', 'click', function(event) {
+        event.stop();
+      });
+      this._element.select('a.move' + this._NEW_TOOL_SELECTOR).invoke('addClassName', this._ENHANCED_TOOL_CSS_CLASS);
+    },
+
+    _syncDefaultValues: function() {
+      this._element.select('.extension-maquette').each((function(el) {
+        el.down('input[name$=default]').disabled = !!!el.up('#default');
```

Comment:
!!! is just a triple negation, and is equivalent to just one !. The only reason to use !!! is to make it stand out as far as I know - http://stackoverflow.com/questions/21154510/the-use-of-the-triple-exclamation-mark